examples/bri-3/src/bri/utils/test.ts

```diff
@@ -0,0 +1,14 @@
+// import Mapper from './mapper';
+// import { getType, Type, PropertyInfo } from "tst-reflect";
+// import { BpiSubjectDto } from './bpiSubject.dto';
+
+// const from  = {
```

Comment:
please make sure to add a few tests

Reply:
will add more tests after this is figured out : https://github.com/eea-oasis/baseline/issues/534